analyzers/tests/SonarAnalyzer.Test/TestCases/TooManyLabelsInSwitch.cs

```diff
@@ -15,15 +17,101 @@ public TooManyLabelsInSwitch(int n, MyEnum en)
             switch (n)
             {
                 case 0:
-                    break;
+                {
+                    {
+                        break;
+                    }
+                }
                 default:
                     break;
             }
 
+            switch(n) // Noncompliant
+            {
+                case 0:
+                    {
+                        break;
+                    }
+                case 1:
+                    {
+                        Console.WriteLine("");
+                        break;
+                    }
+                case 2:
+                    {
+                        {
+                            break;
+                        }
+                    }
+            }
+
+            switch(n) // Noncompliant
+            {
+                case 0:
+                    Console.WriteLine("0");
+                    Console.WriteLine("0+0");
+                    break;
+                case 1:
+                    Console.WriteLine("1");
+                    break;
+                case 2:
+                    Console.WriteLine("2");
+                    break;
+            }
+
+            switch(n) // Noncompliant
+            {
+                case 0:
+                    Console.WriteLine("0");
+                    Console.WriteLine("0+0");
+                    return;
+                case 1:
+                    Console.WriteLine("1");
+                    break;
+                case 2:
+                    Console.WriteLine("2");
+                    break;
+            }
```

Comment:
This duplicates the test above.

Reply:
It has a `return` statement instead of a `break` statement